songpal/notification.py

```diff
@@ -152,6 +152,12 @@ class ContentChange(ChangeNotification, PlayInfo):
 
     make = classmethod(make)
 
+    kind = attr.ib()  # for newer devices
```

Comment:
Add a note that `contentKind` is the preferred way to access this.

Reply:
Happy with a docstring?
![Screenshot 2023-11-04 at 10 38 32 AM](https://github.com/rytilahti/python-songpal/assets/4916781/e785ca4d-2d4a-4eb4-9a2c-69aefcbea762)

Reply:
Looks good to me 👍